index.html

```diff
@@ -1352,37 +1356,20 @@ <h3>Identifiers</h3>
 allows for the expression of statements about specific things in the
 [=verifiable credential=] and is set by an [=issuer=] when expressing
 objects in a [=verifiable credential=] or a [=holder=] when expressing
-objects in a [=verifiable presentation=]. Example `id` values
+objects in a [=verifiable presentation=]. The `id` [=property=] expresses an
+identifier that others are expected to use when expressing statements about a
+specific thing identified by that identifier. Example `id` values
 include UUIDs (`urn:uuid:0c07c1ce-57cb-41af-bef2-1b932b986873`), HTTP URLs
 (`https://id.example/things#123`), and DIDs (`did:example:1234abcd`).
         </p>
 
-        <p>
-<em>If</em> the `id` [=property=] is present:
-        </p>
-
-        <ul>
-          <li>
-The `id` [=property=] MUST express an identifier that others are
-expected to use when expressing statements about a specific thing identified
-by that identifier.
-          </li>
-          <li>
-The `id` [=property=] MUST NOT have more than one value.
```

Comment:
Why is the statement that the `id` must have only one value being deleted?

Reply:
It wasn't deleted, it was collapsed into a more succinct description:

https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1464.html#dfn-id